lib/oidc/handleOAuthResponse.ts

```diff
@@ -81,14 +81,14 @@ export function handleOAuthResponse(sdk: OktaAuth, tokenParams: TokenParams, res
       var accessToken = res.access_token;
       var idToken = res.id_token;
       var refreshToken = res.refresh_token;
-      var now = Math.floor(Date.now()/1000);
 
       if (accessToken) {
         var accessJwt = sdk.token.decode(accessToken);
+        sdk._clock.calculateLocalOffset(accessJwt.payload.iat);
```

Comment:
should local offset be captured if access token is not available?

Reply:
it's captured when either of idtoken or accesstoken is available.

Reply:
ohh, good catch! The idtoken logic was deleted by some reason...